sdk/evaluation/azure-ai-evaluation/README.md

```diff
@@ -0,0 +1,83 @@
+# Azure AI Evaluation
+
+## Introduction
+
+Evaluators are custom or prebuilt classes or functions that are designed to measure the quality of the outputs from language models.
+
+## Usage
+
+Users can create evaluator runs on the local machine as shown in the example below:
+
+```python
+import os
+from pprint import pprint
+
+from promptflow.core import AzureOpenAIModelConfiguration
+
+from azure.ai.evaluation.evaluate import evaluate
```

Comment:
Since the package name is azure.ai.* should this SDK be under the sdk/ai folder, next to azure-ai-inference?

Reply:
We asked DevDiv about this, and they were the ones who suggested to use a separate folder

Reply:
Who are the folks you talked to and what reason did they give? Feel free to IM me if you don't want to answer here. @kristapratico @johanste as FYI.

Reply:
I don't have much context here, adding @annatisch

Reply:
Yeah this one's a bit tricky - we have a lot of packages in the `azure-ai` namespace, the majority of which have their own directory under SDK:
https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/cognitivelanguage
https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/contentsafety
https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/face
https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/documentintelligence
https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/translation

So arguably `inference` should have been under it's own directory as well - though I don't recall being in that discussion so not sure what the reasoning was behind that decision.
Another case could be made that if we want to resurrect the `azure-ai-resources` SDK to be used alongside this one, then maybe they should live side-by-side.
Would customers generally be using this package together with `azure-ai-inference`?

Reply:
The SDKs `\sdk\ai\azure-ai-generative` and `\sdk\ai\azure-ai-resources` are deprecated (someone has the task to start or continue following an official deprecation process). This leaves `\sdk\ai\azure-ai-inference` the only active SDK under `\sdk\ai` folder. 

The evaluations and inference SDKs are likely to be used together.

Perhaps @johanste can chime in on this when he gets back. Anyway, not urgent, we can move folders later if needed.